### ISSUE TITLE
SelectAnnotation not working properly

### ISSUE THREAD

**Issue by sriramreddyM** [recogito/annotorious-openseadragon#99]
annotation is being selected on mouse click sometimes and not sometimes. It was working well before, I observed this recently.
I tried checking in demo, same issue.
https://recogito.github.io/annotorious/getting-started/osd-plugin/

**Comment by sriramreddyM** [comment]
Hi,

I have tested and tried to reproduce this issue with multiple versions that are released. From v2.5.8 and on "SelectAnnotation" is not working properly (It is working fine with till the v2.5.7.). I tested and reproduced the same issue in chrome browser in laptop, ipad, and mobile.

I tried debugging the mouse enter and selectAnnotation events, mouseEnter is always working but not selectAnnotation. 

I can't keep up with the latest releases because of this issue. I have been using a custom widget for vocabulary(a list of categories) as a dropdown, which is working fine in laptops but not on mobiles (only when using openseadragon-annotorious), with the latest releases its seems working in both laptops on mobiles but then the "selectAnnotation" issue arises.

**Comment by rsimon** [comment]
Hm, again, I can't reproduce this. I tried with v2.6.0, standard (not OpenSeadragon), Chrome and Firefox, and the `selectAnnotation` seems to get fired correctly. There must be something additional going on. Do you happen to have a public demo where you can reproduce the effect?

**Comment by sriramreddyM** [comment]
I added it on [fiddle](https://jsfiddle.net/qfs0mrxk/18/,) it is working fine including the drop down selector (in tablets too). But It is still not working on my local machine where I included this annotation tool in a project. 

I'm guessing either there might be something additional going on like conflict with other libraries or it should be weird timing issue that behaves differently at different times.
